asr-python-advanced-tao-ngram-deployment.ipynb

```diff
@@ -0,0 +1,497 @@
+    "You can also work through our tutorial notebooks on [fine-tuning CitriNet](./asr-python-advanced-finetune-am-citrinet-tao-finetuning.ipynb), [fine-tuning CitriNet for noisy audio](./asr-python-advanced-finetune-am-citrinet-for-noisy-audio-withtao.ipynb), and [deploying CitriNet](./asr-python-advanced-finetune-am-citrinet-tao-deployment.ipynb). \n",
+    "- Weighted Finite State Transducer (WFST) tokenizer and verbalizer files for Inverse Text Normalization (ITN). <br> \n",
+    "For more information on WFST and ITN, see this paper: [NeMo Inverse Text Normalization: From Development to Production](https://arxiv.org/pdf/2104.05055.pdf). <br>\n",
+    "You can download pretrained WFST ITN model files from this [NVIDIA GPU Cloud (NGC) model page](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/inverse_normalization_en_us). \n",
+    "- a decoder vocabulary file. You can download one from the [Riva ASR LM NGC model page](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/models/speechtotext_en_us_lm). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "---\n",
+    "## Riva ServiceMaker\n",
+    "Riva ServiceMaker is a set of tools that aggregates all the necessary artifacts (models, files, configurations, and user settings) for Riva deployment to a target environment. It has two main components:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Riva-build\n",
+    "\n",
+    "This step helps build a Riva-ready version of the model. Its only output is an intermediate format (called an RMIR) of an end-to-end pipeline for the supported services within Riva. Let's consider an ASR n-gram language model. <br>\n",
+    "\n",
+    "`riva-build` is responsible for the combination of one or more exported models (`.riva` files) into a single file containing an intermediate format called Riva Model Intermediate Representation (`.rmir`). This file contains a deployment-agnostic specification of the whole end-to-end pipeline along with all the assets required for the final deployment and inference. For more information, refer to the [documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/service-asr.html#pipeline-configuration)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# IMPORTANT: UPDATE THESE PATHS \n",
```

Comment:
is this a TODO?

Reply:
That's an instruction to the user. I've provided placeholder strings on the right side of the assignments in that cell, but none of them is a path to anything. For obvious reasons, I wouldn't know what those paths are on the user's system.

Reply:
Okay, makes sense.  Just wanted to call it out in case it was a leftover internal note.  Please ignore my comment then.